docs/components/Card.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { useState } from "react";
+
+const Card = ({ title, text, iconSVG, link }) => {
+  const [hover, setHover] = useState(false);
+
+  const styles = {
+    cardBackground: {
+      display: "flex",
+      padding: "16px",
+      alignItems: "center",
+      gap: "12px",
+      flex: "1 0 0",
+      borderRadius: "4px",
+      border: "1px solid rgba(0, 0, 0, 0.08)",
+      transition: "background-color 0.3s, color 0.3s",
+      backgroundColor: hover ? "#F2F0EE" : "#FAFAF9",
+    },
+    cardContent: {
+      display: "flex",
+      flexDirection: "column",
```

Comment:
<img width="1159" alt="image" src="https://github.com/latticexyz/mud/assets/508855/a00a8950-f62e-4a8f-bd41-8749129431d9">

https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction

Weird... not sure what's going on here.

Reply:
Ohh, I think because `FlexDirection` is expecting a string const (one of a set of values) rather than an arbitrary string. Try adding `as const` just before the semicolon, e.g.
```ts
const style = {
  ...
} as const;
```